src/LibraryManagementSystem.java

```diff
@@ -105,4 +107,40 @@ public  static  boolean  checkBook(String ISBN){
     public static void countTotalBooks(){
         System.out.println("Total number of the books: " + bookQuantity);
     }
+
+
+	/*
+	   * This method allows returning the given book.
+       * If the book has been returned, it prints a message stating this.
+	*/
+	public  static  void returnBook(String userID,String tittle,String author,String ISBN,String pageNumber){
+		boolean isBookReturned=false;
+		for(int i =0; i<transactionQuantity;i++){
+			if(transactions[i][0].equals(userID)&& transactions[i][1].equals(ISBN)){
+				isBookReturned =true;
+				break;
+			}
+		}
+		String [][] transactionClone=new String[transactionQuantity - 1][3];
+		for(int i = 0; i<transactionQuantity;i++){
+			if(transactions[i][0].equals(userID)&& transactions[i][1].equals(ISBN)){
+				continue;
+			}
+			transactionClone[i][0]=transactions[i][0];
+			transactionClone[i][1]=transactions[i][1];
+			transactionClone[i][2]=transactions[i][2];
+		}
+
+		addBook(tittle,author,ISBN,pageNumber);
+		
```

Comment:
please check the returning date there is a method called `checkBookReturnDeadline(String userId.)` you can use this method